optaweb-vehicle-routing-docs/src/main/asciidoc/development-guide.adoc

```diff
@@ -69,91 +69,50 @@ To run the back end in development mode, enter the following command:
+Use it to permanently store the default configuration and to define Quarkus profiles.
 
-[NOTE]
-It is not possible to set properties by specifying `-D` when running the application using the Spring Boot Maven plugin (`./mvnw spring-boot:run -Dmy-property`).
-Any system properties that should be set by the plugin to the forked Java process in which the application runs need to be specified using `systemPropertiesVariables`
-https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/html/#run-example-system-properties[plugin configuration].
+==== System properties
 
-You can learn more about configuring a Spring Boot application on the
-https://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#boot-features-external-config[Spring Boot Externalized Configuration] page.
+To override the default configuration temporarily, use system properties (`-Dproperty=value`).
 
-[TIP]
-Use `src/main/resources/application-local.properties` to store your personal configuration without affecting the Git working tree.
+==== The `.env` file
+
+To override the default configuration permanently, for example to store configuration that is specific for your development environment, use the `optaweb-vehicle-routing-backend/.env` file.
+This file is excluded from version control and so it does not exist when you clone the repository.
+Use `optaweb-vehicle-routing-backend/.env-example` to initialize your own `optaweb-vehicle-routing-backend/.env` file.
+You can make changes in the `.env` file without  affecting the Git working tree.
 
 See the complete list of <<appendix-backend-config#backend-configuration-properties>>.
 
-See also the complete list of
-https://docs.spring.io/spring-boot/docs/current/reference/html/appendix-application-properties.html[common application properties]
-available in Spring Boot.
+See also the complete list of https://quarkus.io/guides/all-config[common application properties] available in Quarkus.
 
 === Logging
 
 OptaWeb uses the SLF4J API and Logback as the logging framework.
-The Spring environment enables you to configure most logging aspects including levels, patterns, and log files in the same way as any other <<backend-configuration>> (most often using `application.properties` or arguments `--property=value`).
-See the
-https://docs.spring.io/spring-boot/docs/current/reference/html/spring-boot-features.html#boot-features-custom-log-levels[Spring Boot Logging]
-documentation for more information.
-
-Following are examples of properties you can use to control logging level of some parts of the application:
-
-- Use `logging.level.org.optaweb.vehiclerouting=debug` to enable debug level for the back end code.
-- Use `logging.level.org.optaplanner.core=warn` to reduce OptaPlanner logging.
-- Use `logging.level.org.springframework.web.socket=trace` to access more details when investigating problems with WebSocket connection.
+See https://quarkus.io/guides/logging[Quarkus - Configuring Logging] for more information.
```

Comment:
Not major, but we generally use this construction:
For more information , see....